CONTRIBUTING.md

```diff
@@ -1,18 +1,126 @@
+
+A quest is officially marked as accepted, when it get's the ["new quest" tag](https://github.com/westnordost/StreetComplete/labels/new%20quest).
+<!-- TODO: introduce a better way/new tag here? ref https://github.com/westnordost/StreetComplete/issues/1006 -->
+
+## Improving documentation
+
+- ☑️ requires no programming knowledge
+- ⬜️ requires no OpenStreetMap knowledge
+
+You can also help to keep the OpenStreetMap wiki **up-to-date** for StreetComplete. For example:
+* Add missing quests in [the quest list](https://wiki.openstreetmap.org/wiki/StreetComplete/Quests) or check, that they are up to date.
+* Edit [the FAQ](https://wiki.openstreetmap.org/wiki/StreetComplete/FAQ) and add reoccuring questions.
+* Edit [the main StreetComplete page](https://wiki.openstreetmap.org/wiki/StreetComplete).
+
+## Development
+
+- ⬜️ requires no programming knowledge
+- ☑️ requires no OpenStreetMap knowledge (depends on what you do)
+
+If you would like to help and are able to contribute code, you are most welcome.
+
+There are many reasonable feature requests and ideas for new question types in the issue tracker which you could also engage yourself with. If you have own ideas how to improve this app and want to make sure that the Pull Request will be merged, it is strongly suggested to **open an issue first** to discuss the feature.
+
+Note StreetComplete also uses [some dependencies](#issues-of-dependencies), where contributions are likely also accepted and help StreetComplete, too.
+
+If you need to find things, where help is appreciated [have a look at the issues tagged with "help wanted"](https://github.com/westnordost/StreetComplete/labels/help%20wanted).
+
+### Developing new quests
+
+You want to contribute a new quest right away? That's great!
+However, we strongly suggest to [**open an issue** discussing the quest](#suggesting-new-quests), before creating a PR, so we can see if your quest idea meets the criteria and can be included in StreetComplete.
+
+In case you **don't have an idea of a quest?** Look [at the existing issues](https://github.com/westnordost/StreetComplete/issues?q=is%3Aissue+is%3Aopen+label%3A%22new+quest%22+sort%3Areactions-%2B1-desc). Sorted by 👍 reactions you can also see which are the most requested quests.
+
+To start developing you can just download [Android Studio](https://developer.android.com/studio/index.html) and create your quest. It should automatically handle indentation through the [editorconfig file](.editorconfig). If you use another IDE, you may need to [install a plugin](http://editorconfig.org/#download).
+In any case pay attention to [the new quest guide](https://github.com/westnordost/StreetComplete/wiki/Adding-new-Quests-to-StreetComplete) of StreetComplete! It also has tips for implementing a quest.
+
+Beginners can also [use the quest generator](https://ent8r.github.io/StreetCompleteQuestCreator/) for creating simple quests.
```

Comment:
Actually, I would rather not recommend this.

Reply:
Might be better... The only thing you can do with this is creating a yes/no quest but nothing more complex...

Reply:
Okay, unless @ENT8R is not disappointed then, I can do so, of course. Just as you did the work to create that generator, I thought you may wanted to see it being mentioned. But I remove it.